data_steward/validation/participants/identity_match.py

```diff
@@ -618,70 +620,78 @@ def _get_date_string(dataset):
     return datetime.now().strftime(consts.DRC_DATE_FORMAT)
 
 
-def write_results_to_site_buckets(project, validation_dataset=None):
+def write_results_to_site_buckets(storage_client, validation_dataset):
     """
     Write the results of participant matching to each site's bucket.
 
-    :param project: a string representing the project name
+    :param storage_client: a client to generate a bucket for a blob
     :param validation_dataset:  the identifier for the match values
         destination dataset
 
     :return: None
     :raises:  RuntimeError if validation_dataset is not defined.
     """
     LOGGER.info('Writing to site buckets')
-    if validation_dataset is None:
-        LOGGER.error('Validation_dataset name is not defined.')
-        raise RuntimeError('validation_dataset name cannot be None.')
+    if not validation_dataset:
```

Comment:
+1 for Pythonic

Reply:
The real question is which ones get printed:

```
print('using is')
if '' is None:
    print("'' is none")

if [] is None:
    print('[] is None')

print('using not')
if not []:
    print('not []')

if not '':
    print("not ''")

print("we're done here")
```